etna/models/mixins.py

```diff
@@ -629,25 +638,152 @@ def get_model(self) -> Any:
         return self._base_model.get_model()
 
 
-class SaveNNMixin(SaveMixin):
-    """Implementation of ``AbstractSaveable`` torch related classes.
+def _load_object(class_name, class_parameters):
+    cls = get_factory(class_name)
+    obj = cls(**class_parameters)
+    return obj
+
+
+def _save_pl_model(archive: zipfile.ZipFile, filename: str, model: "LightningModule"):
+    with archive.open(filename, "w", force_zip64=True) as output_file:
+        to_save = {
+            "class": BaseMixin._get_target_from_class(model),
+            "hyperparameters": dict(model.hparams),
```

Comment:
Is it possible that `save_hyperparameters` would not be called before?

Reply:
There is a possibility that some user that implemented his own NN from `DeepBaseNet` forgot to save hyperparameters with `self.save_hyperparameters()`. In that case loading should fail.

But in our own classes we make sure that it can be loaded from the disk.

Do you suggest to somehow clarify in the docs that `save_hyperparameters` should be called?

Reply:
Yes, we can describe that this method is expected to be called.